lib/src/layout/scaffold.dart

```diff
@@ -1,118 +1,331 @@
-import 'package:macos_ui/src/library.dart';
+import 'dart:math' as math;
+import 'dart:ui';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
 import 'package:macos_ui/macos_ui.dart';
-import 'package:split_view/split_view.dart';
+import 'package:macos_ui/src/layout/content_area.dart';
+import 'package:macos_ui/src/layout/resizable_pane.dart';
+import 'package:macos_ui/src/layout/resizable_pane_notifier.dart';
+import 'package:macos_ui/src/layout/sidebar.dart';
+import 'package:macos_ui/src/layout/title_bar.dart';
+
+const _kTitleBarHeight = 52.0;
+
+const _kSmallTitleBarHeight = 30.0;
 
 /// A basic screen-layout widget.
 ///
 /// Provides a [body] for main content and a [sidebar] for secondary content
```

Comment:
Body seems to have been replaced by `children`, so this documentation needs to be updated.